Assets/Scripts/Manager/ThemeMusicManager.cs

```diff
@@ -14,15 +14,18 @@ public class ThemeMusicManager : MusicManager {
 	public AudioSource mainTheme;
 	public AudioSource mechanicalTheme;
 	public AudioSource choirTheme;
+    public AudioSource titleTheme;
 
 	//Song Volume
 	public float mainThemeVolume;
 	public float mechanicalThemeVolume;
 	public float choirThemeVolume;
+    public float titleThemeVolume;
 
 	private static String[] mainThemeScenes = {"OpenScene"};
 	private static String[] mechanicalThemeScenes = {"1a", "1b", "2a", "2b", "3a", "3b"};
 	private static String[] choirThemeScenes = {"4a-i", "4a-ii", "4b", "5a", "5b"};
+    private static String[] titleThemeScenes = { "TitleScreen"};
```

Comment:
if we know it's just gonna be used for title, might as well make it a sting instead of string []
I use [] for openScene cos I was thinking the main theme might be played somewhere else